plugins/transformations/seq2array/doc/introduction.rst

```diff
@@ -0,0 +1,15 @@
+.. include:: vars.rst
+
+.. _section-introduction:
+
+************
+Introduction
+************
+
+|RS_SEQ2ARRAY_TSFM_TM| provides a way of transforming DDS samples that contain
+sequences into DDS samples that uses arrays.
+
+The plugin supports any input data type, and it checks that the output datatype
+is compatible (just replacing sequences by arrays). It also checks that the size
+of the corresponding sequences fit into the array, otherwise it will throw an
+error.
```

Comment:
the 'compatibility' of different types is done once at the beginning. However, the size check is done at runtime, since the transformation cannot know the number of the sequence elements till it receives data.